ui/app/AppLayouts/Onboarding2/KeycardCreateProfileFlow.qml

```diff
@@ -123,9 +124,17 @@ SQUtils.QObject {
     Component {
         id: backupSeedRevealPage
         BackupSeedphraseReveal {
-            seedWords: root.seedWords
+            Component.onCompleted: {
+                try {
+                    const seedwords = root.getSeedWords()
+                    d.seedWords = JSON.parse(seedwords)
+                    root.seedphraseSubmitted(d.seedWords)
+                } catch (e) {
+                    console.error('Failed to get seedwords', e)
+                }
+            }
+            seedWords: d.seedWords
 
-            onMnemonicWasShown: root.mnemonicWasShown()
```

Comment:
Needed I guess?

Reply:
Not needed in the end since status-go gets rid of it.